checks/readme.php

```diff
@@ -69,6 +69,33 @@ function check_for_warnings() {
 		}
 	}
 
+	public function check_tested_up_to() {
```

Comment:
```suggestion
	public function check_tested_up_to() {
		// Bail early.
		if ( empty( $this->readme->tested ) ) {
			return;
		}
```

This avoids the HTTP request unless actually needed.